ios/Classes/HyperWebviewFlutterPlugin.m

```diff
@@ -0,0 +1,106 @@
+#import "HyperWebviewFlutterPlugin.h"
+
+@interface HyperWebviewFlutterPlugin()
+
+@property (nonatomic, strong) FlutterMethodChannel* dartChannel;
+@property (nonatomic, strong) FlutterMethodChannel* nativeChannel;
+
+@end
+
+const int OPENAPPS_REQUEST_CODE = 19;
+const int GET_RESOURCE_NAME = 453;
+const int CAN_OPEN_APP_CODE = 789;
+@implementation HyperWebviewFlutterPlugin
+
+
++ (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar>*)registrar {
+    HyperWebviewFlutterPlugin* instance = [[HyperWebviewFlutterPlugin alloc] init];
+    instance.dartChannel=[FlutterMethodChannel methodChannelWithName:@"DartChannel" binaryMessenger:[registrar messenger]];
+    FlutterMethodChannel* channel = [FlutterMethodChannel methodChannelWithName:@"NativeChannel" binaryMessenger:[registrar messenger]];
+    instance.nativeChannel=channel;
+    [registrar addMethodCallDelegate:instance channel:channel];
+}
+
+- (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
+  if ([@"openApp" isEqualToString:call.method]) {
+      [self openApp:[call.arguments objectAtIndex:1]];
```

Comment:
Let's use firstObject only

Reply:
@Subhash703 - to take care in next PR if still open